srml/support/procedural/tools/src/syn_ext.rs

```diff
@@ -72,7 +72,7 @@ groups_impl!(Braces, Brace, Brace, parse_braces);
 groups_impl!(Brackets, Bracket, Bracket, parse_brackets);
 groups_impl!(Parens, Paren, Parenthesis, parse_parens);
 
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
hmm I now remember I introduced this Clone before creating the new enum HasherKind. So I very suspect this change is not needed anymore.

Reply:
As we will remove this anyway, it is no problem.